templates/result-email-template.html

```diff
@@ -72,13 +98,23 @@ <h2>General Information</h2>
             <td>{{ log_link }}</td>
         </tr>
         <tr>
-            <th style="color: blue">Total suite duration</th>
-            <td>{{ suite_run_time }}</td>
+            <th style="color: blue">Invoked by User</th>
+            <td>{{ trigger_user }}</td>
+        </tr>
+    </table>
+    <h3>Suite duration</h3>
+    <table class="half-table">
+        <tr>
+            <th style="color: blue">Start Time</th>
+            <td>{{ suite_run_time.start }}</td>
         </tr>
+        <tr>
+            <th style="color: blue">End Time</th>
+            <td>{{ suite_run_time.end }}</td>
         </tr>
         <tr>
-            <th style="color: blue">Invoked by User</th>
-            <td>{{ trigger_user }}</td>
+            <th style="color: blue">Total Run Time</th>
```

Comment:
Let's keep it as `Duration` | `Execution Duration`

Reply:
Changed to Execution Duration